src/usr/local/www/system_usermanager_settings.php

```diff
@@ -177,15 +174,19 @@
 
 $section = new Form_Section('Settings');
 
+list($d_hr, $d_min) = [int($g['default_session_timeout_mins']/60), $g['default_session_timeout_mins'] % 60];
```

Comment:
This seems like a lot of extra work for what it does. I'd just express it all in minutes since that's what the field takes.